apps/public-docsite-resources/config/api-docs.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   apiJsonPaths: [
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28166&commitId=ec7826126dac67f01859c45f81570c7faccea819&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas - RTL.default.chromium.png|0| Removed|
|ActivityItem.Personas.default.chromium.png|0| Removed|
|ActivityItem.Root - RTL.default.chromium.png|0| Removed|
|ActivityItem.Root.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>